deployments/deployments.json

```diff
@@ -1,7 +1,164 @@
 [
   {
-    "name": "Ethereum Mainnet",
-    "chainId": 1,
-    "url": ""
+    "name": "Sepolia",
+    "chainId": 11155111,
+    "url": "https://sepolia.etherscan.io/address/0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed",
+    "sourcify": "https://repo.sourcify.dev/contracts/partial_match/11155111/0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed/"
```

Comment:
Do we use this key anywhere? Curious to hear the thinking behind it, e.g. should we accept PRs that add additional block explorer URLs? Maybe we use an array of `urls` instead, and the frontend can default to the first one but allow you to view the others also

Reply:
Well, the reasoning is more the following: the _usual_ setup is that you have at the beginning a Blockscout instance and later you either have an additional Etherscan instance or even discontinue the Blockscout instance. Standard Blockscout instances automatically verify contracts that are Sourcify verified (cool feature IMHO). Having the Sourcify link ensures anyone understands that a potential additional Blockscout instance is already verified without any action. Thus, it's just an additional piece of meta-information I deem important. The reason why I want to have it as part of the `deployments.json` file is that maybe later I want to incorporate it into the UI in some way. Regarding your question: one URL seems fine to me; since I automatically verify the URLs in the CI we see if an existing URL is discontinued and thus can act accordingly. Will resolve this comment for now, since I'm gonna merge the PR, but please continue the discussion if you think my thoughts are stupid :)

Reply:
> Sourcify link ensures **anyone understands** that a potential additional Blockscout instance is already verified without any action

I think most people will not be familiar with this feature (I didn't know about it until this message) 😅

> The reason why I want to have it as part of the deployments.json file is that maybe later I want to incorporate it into the UI in some way.

To this point though, wouldn't a `"urls":["https://etherscan...", "https://sourcify...."],` style be simpler to manage there? We don't need to worry about other keys like blockscout, https://parsec.fi, etc and just add them all in the array

Reply:
> I think most people will not be familiar with this feature (I didn't know about it until this message) 😅

Lol, I'm just too much into verification 😄 

> To this point though, wouldn't a "urls":["https://etherscan...", "https://sourcify...."], style be simpler to manage there? We don't need to worry about other keys like blockscout, https://parsec.fi/, etc and just add them all in the array

Ok, we can do this. Will adjust in the soonish follow-up prod deployment PR.

Reply:
Addressing this comment via https://github.com/pcaversaccio/createx/pull/57.